criu/proc_parse.c

```diff
@@ -1190,7 +1190,7 @@ struct opt2flag {
 
 static bool sb_opt_cb(char *opt, char *unknown, size_t *uoff)
 {
-	unsigned int id;
+	int id;
```

Comment:
If you change a variable type, you need to create a separate patch and explain in a commit message why you think it has to be done. In this case, I don't think the change is correct. uid and gid are uint32.

Reply:
Instead of typecasting, I thought I will change the variable type of id since it is a local variable and that was the mistake. I will use typecasting instead.